### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Poussinou**
This small PR adds two buttons to get the app in the README.md

Thanks for this app :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@
 
 _Browse like no one’s watching. The new Firefox Focus automatically blocks a wide range of online trackers — from the moment you launch it to the second you leave it. Easily erase your history, passwords and cookies, so you won’t get followed by things like unwanted ads._
 
+<a href="https://f-droid.org/packages/org.mozilla.klar/" target="_blank">
+<img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="90"/></a>
+<a href="https://play.google.com/store/apps/details?id=org.mozilla.focus" target="_blank">
+<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="90"/></a>
+
```
Comment (pocmo) [inline README.md]: This should use MarkDown syntax instead of HTML.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,11 @@
 
 _Browse like no one’s watching. The new Firefox Focus automatically blocks a wide range of online trackers — from the moment you launch it to the second you leave it. Easily erase your history, passwords and cookies, so you won’t get followed by things like unwanted ads._
 
+<a href="https://f-droid.org/packages/org.mozilla.klar/" target="_blank">
+<img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="90"/></a>
```
Comment (pocmo) [inline README.md]: The builds on F-Droid are not maintained by us - So I do not feel comfortable adding this badge to our readme.

Comment (Poussinou) [inline README.md]: F-Droid contains only FLOSS apps, yours is welcome, but if you want me to remove this badge, I'll do ;)

File: README.md (hunk 1 above)
Comment (Poussinou) [inline README.md]: I cannot resize the badge using MarkDown syntax. If you find interesting informations about this, please share, I am very interested :)

File: README.md (hunk 2 above)
Comment (pocmo) [inline README.md]: Yes, please!

File: README.md (hunk 1 above)
Comment (pocmo) [inline README.md]: Ah, I see. The badge is indeed massive.

Comment (Poussinou) [inline README.md]: Yep, they are very massive

**Comment by Poussinou** [comment]
Please review now :smiley:

**Pull request closed without merging**